packages/codemods/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@fluentui/codemods",
+  "version": "0.1.0",
+  "description": "package to enable easy upgrades to new versions",
+  "private": true,
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "bin": {
+    "upgrade": "./bin/upgrade.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@types/jest": "~24.9.0",
+    "@types/node": "^10.3.2",
+    "@uifabric/build": "^7.0.0",
+    "@uifabric/tslint-rules": "^7.2.2"
+  },
+  "dependencies": {
+    "tslib": "^1.10.0",
+    "@types/react": "16.8.25",
+    "@types/semver": "^5.5.0",
+    "acorn": "^6.4.1",
+    "minimist": "^1.2.2",
+    "office-ui-fabric-react": "^7.121.7",
```

Comment:
If this is only used in tests, it should be a dev dependency.

Reply:
It is used in real code, at least for now.